src/System.IO.Pipelines/System/IO/Pipelines/Pipe.cs

```diff
@@ -144,11 +150,10 @@ WritableBuffer IPipeWriter.Alloc(int minimumSize)
                 }
 
                 _currentWriteLength = 0;
-                return new WritableBuffer(this);
             }
         }
 
-        internal void Ensure(int count = 1)
+        void IPipeWriter.Ensure(int count)
```

Comment:
What motivated the removal of the default arg value?

Reply:
This would only be called via interface and interface method's default value would always be used.